eng/pipelines/runtime-staging.yml

```
@@ -99,6 +99,34 @@ jobs:
     browser: firefox
     alwaysRun: ${{ variables.isRollingBuild }}
 
+#
+# Build the whole product using Mono
+#
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/common/global-build-job.yml
+    helixQueuesTemplate: /eng/pipelines/libraries/helix-queues-setup.yml
+    buildConfig: Release
+    runtimeFlavor: mono
+    platforms:
+    - FreeBSD_x64
+    variables:
+      # map dependencies variables to local variables
+      - name: librariesContainsChange
+        value: $[ dependencies.evaluate_paths.outputs['SetPathVars_libraries.containsChange'] ]
+      - name: monoContainsChange
+        value: $[ dependencies.evaluate_paths.outputs['SetPathVars_mono.containsChange'] ]
+    jobParameters:
+      testScope: innerloop
+      nameSuffix: AllSubsets_Mono
+      buildArgs: -s mono+libs+host+packs -c $(_BuildConfig) /p:ArchiveTests=true /p:UseLiveBuiltDotNetHost=true
```

```suggestion
      buildArgs: -s mono+libs+host+packs -c $(_BuildConfig)
```
This is not doing any testing, so these args should not be needed.